packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -512,7 +513,7 @@ final TextTreeConfiguration denseTextConfiguration = TextTreeConfiguration(
 ///    ╚═══════════
 /// ```
 ///
-/// /// See also:
+ /// See also:
```

Comment:
nit: Looks like there may still be an extra space here at the beginning of the line?

Reply:
Oooh, good catch.  Fixed.